docker/scripts/osie.sh

```diff
@@ -542,8 +542,14 @@ if [[ $pwhash == "preinstall" ]]; then
 	exit 0
 fi
 
-echo -e "${GREEN}#### Setting root password${NC}"
-set_root_pw "$pwhash" $target/etc/shadow
+echo -e "${GREEN}#### Setting password${NC}"
+if [[ ${OS} =~ vmware_nsx_3_0_0 ]] ; then
+	pwuser=admin
+	set_pw "$pwuser" "$pwhash" $target/etc/shadow
+else
+	pwuser=root
+	set_pw "$pwuser" "$pwhash" $target/etc/shadow
+fi
```

Comment:
All we need in the if state is pwuser and not the call to `set_pw`, lets do this instead:
```sh
pwuser=root
if [[ ${OS} =~ ^vmware_nsx ]]; then
    pwuser=admin
fi
set_pw pwuser "$pwhash" $target/etc/shadow
```